docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -806,4 +806,163 @@ The following is performed on ArduCopter version v4.5.7
+independent motors. Each individual command requires 14 bits, or 1.75 bytes. To transmit 8, we'll require 14 bytes. With the 2 byte CRC, we need to send a total of 16 data 
+bytes (ignoring tail bytes). Now, we need to figure out how many full, 7 data byte, frames we will have in sending 16 bytes. We find this by:
+
+floor(16 bytes * (1 frame / 7 bytes)) = 2 full frames
+
+Then, we need to find how much data remains which we can find by taking:
+
+16 bytes - (2 full frames * 7 bytes/frame) = 2 data bytes remaining.
+
+For this last frame, we need to account for the tail byte since we're not sending a full frame, so we'll have 3 total bytes of data in this final frame. Finding the total 
+number of bits is then done by:
+
+(131 bit/full frame * 2 full frame) + (67 bit/empty frame + (3 data bytes * 8 bit/byte)) = 353 bits
+
+This example is visualized in the DroneCAN GUI Tool's bus monitor here:
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/multi_frame_raw_cmd.png
+
+Now, we can once again find our utilization of 1 second at 500kbit by:
+
+353 bit * 1/500Kbps * 100% = 0.071% Utilization
+
+We can generalize utilization of RawCommands of more than 4 unique ESCs as the following steps:
+
+* Bytes for All Commands (C) = 1.75 bytes/command * (Number of Commands) + 2 bytes CRC
+* Number of full frames (F) = floor(C / 7)
+* Total bytes in last  frame (L) = C - 7F + 1 
+* Total bits (b) = 131 bitfull frame * F + (67 bit/empty frame + 8L)
+* Utilization of 1 Second (%) = b * 1/bitrate * Command Frequency (Hz) * 100%
+
+For 6 commands:
+
+.. csv-table:: Six Raw Command Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/six_raw_cmds.csv
+	:header-rows: 1
+
+For 8 commands:
+
+.. csv-table:: Eight Raw Command Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/eight_raw_cmds.csv
+	:header-rows: 1
+
+DroneCAN Utilization of a Quadcopter
+======================================
+Now that we know how to calculate the utilizations of various DroneCAN messages, it is easy to extend our understanding to an entire vehicle of DroneCAN nodes. In order to do 
+so, we simply need to sum together the impacts of each message. Let's suppose that our flight controller transmits RawCommand messages at 400Hz, and that each of our modules 
+transmits telemetry at 10Hz.
+
+Since we are controlling four modules, we'll need four commands per RawCommand message, and we'll need only one frame to send them. Using our calculation :ref:`above <raw_cmd_utilization>`, 
+we find that the utilization of these commands are:
+
+.. csv-table:: Quad Example RawCommand Bus Utilization
+	:file: ../_static/comms_protocols_pictures/dronecan/quad_ex_raw_cmd.csv
+	:header-rows: 1
+
+Then, we can use our :ref:`telemetry calculation <telemetry_utilization>` in order to find the impact of the four modules reporting telemetry as:
```

Comment:
I think in the rest of this chunk you've added you tend to call them motors, but here its modules. I guess maybe stick with one term for this chunk at least